drivers/clock_control/stm32_ll_clock.c

```diff
@@ -198,6 +226,31 @@ static void stm32_clock_switch_to_hsi(u32_t ahb_prescaler)
 	}
 }
 
+/*
+ * MCO configure doesn't active requested clock source,
```

Comment:
Then we should activate source clock in stm32_clock_control_mco_init,
at least a check clock is enabled would be required.